ShristiC/wellness-vitality.github.io
# Bugfix/Fix website UI based on design review

Website fixes
- Our stroke smaller
- Title a little bigger
- Navigation font a little bigger
- Fix arrow on the resonate section
- Functional approach
    - Increase padding between title and text
    - Reduce text width to 80%
    - Reduce padding overall
- specialties: increase title font
- Pillars use exported with fonts
- Icon go down ideally towards center
- testimonial: space between text and frame
- Add some padding to footer
- About: tabs and hamburger
- qualifications: increase padding in between
- Improve actions tag
- gallery: full box with black image. Rounded corners
- Testimonial cover: increase padding between text
- Bring up all the text
- Make testimonial card same size and longer
- Change gray color: #ebeced. 100% opacity
- Reduce quote icon size
- Anonymous person image
- See more text needs to be bigger: at least same size as main content
